discourse_evaluator.py

```diff
@@ -1,8 +1,11 @@
+from dc_eval_models.bert_base import BertBaseModel
+
+
 class DiscourseEvaluator:
     def __init__(self, weights) -> None:

```

Comment:
I added `weights` as an argument for the constructor to encourage passing the weights path instead of hard-coding it inside the class definition. But if you decide not to follow that route, please change the arguments list accordingly.